### ISSUE TITLE
Can't use release for private repository

### ISSUE THREAD

**Issue by praveenpuglia** [zeit/release#139]
Hi!

I have used release before and it works perfectly fine for public repositories. But for private repositories I always face the following.

```
❯ release patch
✔ Bumped version tag to 2.0.1
✔ Created release commit
✔ Tagged commit
✔ Pushed everything to remote
Error! Could not determine GitHub repository.
```
Here's what the package.json looks like.
```json
{
  "name": "joveo-tars",
  "version": "2.0.1",
  "description": "Setup for pristine Joveo Machines",
  "main": "main.js",
  "repository": "git@github.com:joveo/tars.git",
  "scripts": {
    "start": "node main.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "private": true,
  "author": "Praveen Puglia",
  "license": "MIT",
  "dependencies": {
    "chalk": "^1.1.3",
    "cli-spinner": "^0.2.6",
    "download": "^5.0.3",
    "inquirer": "^3.0.6",
    "jsonfile": "^2.4.0",
    "prettyjson": "^1.2.1",
    "shelljs": "^0.7.7"
  }
}
```

I am not sure why this error comes up - `Error! Could not determine GitHub repository`. Does release need some Auth token from github in order to work for private repos?

**Comment by joaolavoier-luizalabs** [comment]
Hi @praveenpuglia. Have you solved this problem? I am looking for a tool like this, but I would like to know if it will work for me. I am also working on a private repository.

**Comment by praveenpuglia** [comment]
@joaolavoier-luizalabs nope. It still doesn't work for private repositories. I don't know if release needs a  special github token to work with private repositories. Haven't got any help from any team member here too. So no idea.

**Comment by praveenpuglia** [comment]
I haven't tried it with latest release so no idea. will do and let you all know if it works.

**Comment by whollacsek** [comment]
I randomly found out the solution just last week. In my case I migrated my repo into an org, which means the `release` app is missing permissions from the org. This link might work for you https://github.com/settings/connections/applications/08bd4d4e3725ce1c0465

**Comment by praveenpuglia** [comment]
I see why this is going to work but is it safe to give it the private access because this seems scary!

> Full control of private repositories

> Applications act on your behalf to access your data based on the permissions you grant them. Organizations control which applications are allowed to access their private data. Applications you authorize will always have access to public data in your organizations. Read about third-party access.

**Comment by Stanzilla** [comment]
Yeah wish there was a better solution to this
